src/components/AddAdvert/AddAdvertForm/AddAdvertForm.jsx

```diff
@@ -31,6 +60,17 @@ const AddAdvertForm = () => {
     };
 
 
+    const handleCategoryChange = (e) => {
+        const category = e.currentTarget.value;
+
+        if (category !== '') {
+            setIsChoosenCategory(true);
+        } else {
+            setIsChoosenCategory(false);
+        }
```

Comment:
setIsChoosenCategory(category !== '');

Btw, there grammatically it has to be setIsChosenCategory (o instead of oo).

Reply:
has fixed